app/policies/recipe_policy.rb

```diff
@@ -1,6 +1,7 @@
 # frozen_string_literal: true
 
 class RecipePolicy < ApplicationPolicy
+  class Scope < Scope
```

Comment:
Is the inheritance needed here? 🤔

Reply:
Yesh. We're inheriting scope and user from the parent here. :3